src/main/java/callcenter/CallManager.java

```diff
@@ -0,0 +1,79 @@
+package callcenter;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class CallManager {
+
+    private static CallManager instance = null;
+
+    private CallManager(){
+
+    }
+    public static CallManager getInstance() {
+        if(instance ==null){
+            instance = new CallManager();
+        }
+        return instance;
+    }
+    private static int levels = 3;
+    private List<Employee>[] employeesList;
+    private Queue<Call> callQueue;
+    public CallManager(int numberOfFreshers, int numberOfLeads) {
```

Comment:
```suggestion
    private Queue<Call> callQueue;

    public CallManager(int numberOfFreshers, int numberOfLeads) {
```

very tough to read code without this extra line.